src/credentials/FabricTable.h

```diff
@@ -450,14 +479,20 @@ class DLL_EXPORT FabricTable
     ConstFabricIterator begin() const { return cbegin(); }
     ConstFabricIterator end() const { return cend(); }
 
+    void SetListener(FabricListener * fabricListener) { mFabricListener = fabricListener; }
+
 private:
     FabricInfo mStates[CHIP_CONFIG_MAX_FABRICS];
     FabricStorage * mStorage = nullptr;
 
     FabricTableDelegate * mDelegate = nullptr;
 
+    FabricListener * mFabricListener = nullptr;
```

Comment:
How is this different from the FabricTableDelegate we already have?

Reply:
Not that much actually, removing this class to use the delegate instead.